### ISSUE TITLE
Extra NAND GAMES not showing if boot with ES MOD

### ISSUE THREAD

**Issue by henriquewicz** [Project-Lunar/Project-Lunar-Issue-Tracker#14]
When I turn on the genesis mini with the pen drive plugged in, if I want to enter the official gaming interface it appears in the original language (in my case Japanese) and when I change the system does not accept and returns to Japanese. Also it does not show the extra games I had posted on NAND via the desktop application game manager. If I remove the flash drive and turn it on, the official interface will be in English (as I had already configured before) and my extra games will appear. I believe this behavior seems like a bug.

**Comment by DefKorns** [comment]
Hi @henriquewicz ,
How do i put this in a easy  way to understand...
USB and NAND are like 2 "different consoles", you can have different games and languages on both. 
you can have it like this:

**NAND:**
> - added sonic 3
> - added  rambo III
> Sync 
> - change language to English

**USB:**
> - added Sonic & Knuckles
> - added Terminator
> Sync 
> - Change language to Japanese

**Comment by henriquewicz** [comment]
yeah but my bug here was . i added rambo3 at nand, i add terminator in usb. i've boot with usb, ihad the 2 games (
RAMBO AND TERM) + the originals game - i've boot WITHOUT usb - i expect terminator was'nt there, but i expect rambo be there. And this is not happening,. without usb, "the nand console" shows ONLY the original games. the extra games in nand isnt showed. And if i change the region from usa (the console is jap), not works. I've changed but any seconds after, back to JAP. "NAND console" its only and forever in jap. Just "the usb console"accepts be changed to english. Understand now my report? Regards

**Comment by DefKorns** [comment]
Did you sync with the USB plugged in?

**Comment by henriquewicz** [comment]
Yes, Def :)
